tests/main/null/task.yaml

```diff
@@ -0,0 +1,6 @@
+summary: Dummy test case which does nothing
```

Comment:
Indeed.. if we can't pass _any test at all,_ it means the suite isn't working anyway. So we may as well just not have anything.